nextjs-app/components/pages/main/WhatWeBring.tsx

```diff
@@ -0,0 +1,87 @@
+"use client";
+import Button from "@/components/common/Button";
+import Image from "next/image";
+import { useRouter } from "next/navigation";
+
+interface GoalCardProps {
+  imageSrc: string;
+  title: string;
+  subTitle: string;
+  description: string;
+}
+
+const GoalCard = (props: GoalCardProps) => {
+  const { imageSrc, title, subTitle, description } = props;
+  return (
+    <div className="flex flex-col items-center rounded-3 transition-all duration-300 border border-[#7E7059] bg-white z-10">
+      <div className="h-[30px]">
+        <Image
+          src={imageSrc}
+          alt={title}
+          width={120}
+          height={120}
+          className="-translate-y-1/2"
+        />
+      </div>
+      <div className="px-7 pt-11 pb-8 space-y-6">
+        <div className="space-y-2 text-[#7E7059]">
+          <h3 className="text-h4">{title}</h3>
+          <h6 className="text-h6">{subTitle}</h6>
+        </div>
+        <p className="text-3 leading-5">{description}</p>
+      </div>
+    </div>
+  );
+};
+
+const WhatWeBring = () => {
+  const router = useRouter();
+  return (
+    <section className="py-16 bg-[#090E3E]">
+      <div className="max-w-[343px] md:max-w-[672px] lg:max-w-[1156px] mx-auto text-center">
+        <h2 className="text-h4 text-white">核心精神</h2>
+        <h3 className="mb-6 text-[48px] text-white font-medium font-eb-garamond leading-11">
+          What We Bring
+        </h3>
+        <Image
+          src="/images/title-symbol-line-white.png"
+          alt="title-symbol-line"
+          width={57}
+          height={5}
+          className="mx-auto mb-24"
+        />
+        <div className="grid md:grid-cols-3 gap-[80px] md:gap-4 lg:gap-6 place-content-center mb-9">
```

Comment:
Seems the token was adjusted

```suggestion
        <div className="grid md:grid-cols-3 gap-[80px] md:gap-7 place-content-center mb-9">
```

<img width="641" alt="image" src="https://github.com/user-attachments/assets/01b68331-963f-4c78-93ad-5501a7c1bc10" />